install_microshift.py

```diff
@@ -0,0 +1,160 @@
+import subprocess
+import time
+import host
+from logger import logger
+
+def microshift_iso_builder(h: host.Host):
+    cmd = f"rm -rf rhocp-4.14.toml fast-datapath.toml"
```

Comment:
rm, chown, chmod commands, there should be native python functions instead of using bash.